### ISSUE TITLE
Issue with the interception of database operations in EF Core 3.1, cannot access IDbCommandInterceptor

### ISSUE THREAD

**Issue by lamelyan** [dotnet/efcore#19484]
I have installed EF Core 3.1 but when I try to use the `IDbCommandInterceptor` it's not available in the EF Core and asks me to install EntityFramework. 

Here is a screenshot that includes a list of packages I have installed.  When I try to implement `IDbCommandInterceptor`, it tells me that it's only available in EF. 

![ef-db-command-interceptor](https://user-images.githubusercontent.com/1872686/71768220-d5ebc200-2ed9-11ea-9385-f0da0c6e3b62.png)

**Comment by ralmsdeveloper** [comment]
@lamelyan 

Use `DbCommandInterceptor` instead of `IDbCommandInterceptor`

I made a short article about it, it's in Portuguese, but I believe it will be easy for you.
[http://ralms.net/dica/efcore31_operations_interception/](http://ralms.net/dica/efcore31_operations_interception/)

**Comment by ralmsdeveloper** [comment]
More information:
[https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/#interception-of-database-operations](https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/#interception-of-database-operations)

**Comment by lamelyan** [comment]
@ralmsdeveloper thank you for the quick reply.  I tried to use `DbCommandInterceptor` and I get the same message. 
![ef-db-command-interceptor-2](https://user-images.githubusercontent.com/1872686/71769165-1bf95380-2ee3-11ea-87df-01cc9c9273a2.png)

**Comment by ralmsdeveloper** [comment]
> More information:
> https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/#interception-of-database-operations

@ajcvickers
 Any particular reason for your reaction, is there anything else?

**Comment by ralmsdeveloper** [comment]
@lamelyan 

I was able to do a repo here.
MySql.Data.EntityFrameworkCore(8.0.18) Does Not Support EFCore 3.X

![image](https://user-images.githubusercontent.com/16335882/71769478-b4ea9780-2f00-11ea-88dc-6acfced4db1d.png)

**Comment by ajcvickers** [comment]
@lamelyan Try the [Pomelo MySQL provider](https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySql/3.1.0). It supports EF Core 3.1.

**Comment by gustavobmichel** [comment]
Hi @ralmsdeveloper thanks for your sample. Though I feel like it doesn't work with EF Core 3.1.11. Your sample was using version 3.1 and I was able to trigger the command up to 3.1.10. When I upgraded it to 3.1.11, it no longer triggers the interceptor.

I have got the same symptoms on my app that uses 3.1.11. It's strange that it works fine using net framework Web api but not when using net core Web api.
